osdep/threads-posix.h

```diff
@@ -23,6 +23,7 @@
 
 #include "common/common.h"
 #include "config.h"
+#include "osdep/compiler.h"
 #include "timer.h"
```

Comment:
not directly related but isn't this file misusing inline functions a bit too much?
it's more and more headers pulled into every file that needs threads and surely people who care about optimization can afford to use LTO

Reply:
Except two init functions most others are one line which are perfect candidate for inlinining. Whole thing is not even 200 lines. We can always move things to thread-posix.c but I doubt this will bring compilation time increase in fact.

Reply:
I might make a PR sometime, not important.